src/petTrack.js

```diff
@@ -0,0 +1,109 @@
+// PROTOTYPE SYNTAX PET
+
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0; 
+const FITNESS_INCREASE = 4;
+const MINIMUM_HUNGER = 0;
+const REDUCE_HUNGER = 3;
+const ACCEPTABLE_HUNGER = 5
+const ACCEPTABLE_FITNESS = 3
+const MAXIMUM_HUNGER = 10
+const MAXIMUM_AGE = 30
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = MINIMUM_HUNGER;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.fitness > MINIMUM_FITNESS && this.hunger < MAXIMUM_HUNGER && this.age < MAXIMUM_AGE;
+    }
+}
+
+Pet.prototype.growUp = function () {
+    
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive');
+      }  
+    
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
+
+    if (this.fitness < 0) {
+    this.fitness = MINIMUM_FITNESS;
+    }
```

Comment:
This could be refactored a bit. In essence you are removing a value to the existing fitness, without allowing it go under a set limit. You can do this with Math.max, which will return the highest of the values passed to it:
```
    this.fitness = Math.max(this.fitness - 3, MINIMUM_FITNESS);
```
Also, you could have a constant for the fitness decrement as you do for the increment